src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_constants.py

```diff
@@ -3,6 +3,8 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import SkuName
```

Comment:
is 'v2018_02_01....' the default api-version of the sdk being used for the acr module?

Reply:
Nope

Reply:
It looks like `SkuName` hasn't changed since the 2017_10_01 version, so you could do: `from azure.mgmt.containerregistry.models import SkuName` if you so chose. Is there a reason for referencing the newer model instead of the older?

With that, I see [imports](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py#L10) straight from the `v2017_10_01` sdk and other imports like this one from the newer. Is there a method to choosing one vs the other?

Reply:
It doesn't matter. CLI is consistently using `2018-02-01-preview` version while the Python SDK defaults to `2017-10-01`. It's unnecessary confusion to use models from different version in different files.

Reply:
Agreed, thanks for fixing the inconsistency issue.